pkg/virt-handler/vm.go

```diff
@@ -2290,3 +2293,23 @@ func isACPIEnabled(vmi *v1.VirtualMachineInstance, domain *api.Domain) bool {
 		domain.Spec.Features != nil &&
 		domain.Spec.Features.ACPI != nil
 }
+
+func setMissingDomainInterfacesStatusNames(interfacesByName map[string]v1.Interface, interfaceStatusByMac map[string]api.InterfaceStatus) {
```

Comment:
It would have been nicer to pass in the specIfaceNamesByMac and not call it inside. When you pass in such structures, someone may mutate it... or in other words, when looking at the caller, you do not know if it is mutated or not.

I won't insist on this one.

Reply:
you mean to call outside indexSpecIfaceNamesByMac and then to just send the result of it to setMissingDomainInterfacesStatusNames as param ?